projects/hinkal/index.js

```diff
@@ -1,17 +1,43 @@
-const { sumTokensExport } = require('../helper/unwrapLPs')
+const registryTokensByChain = require("./registryTokens.js");
+const { getAllTokenBalances, fetchTotalValue } = require("./hinkalUtils.js");
+const { toUSDTBalances } = require("../helper/balances.js");
 
-const config = {
-  ethereum: "0x2ea81946fF675d5Eb88192144ffc1418fA442E28",
-  arbitrum: "0x41658B0DaF59Bb2FbB2D9A5249207011d2B364De",
-  optimism: "0x41658B0DaF59Bb2FbB2D9A5249207011d2B364De",
-  polygon: "0xeEeeb52E36c78b153caaB2761c369a50b066cDD5",
-  avax: "0x41658B0DaF59Bb2FbB2D9A5249207011d2B364De",
-  bsc: "0x0036E884Cab4F427193839788EDEBB4B92B9a069",
-  base: "0x41658B0DaF59Bb2FbB2D9A5249207011d2B364De",
-}
+const tvl = async (_, _1, _2, { chain }) => {
+  const tokenBalances = await getAllTokenBalances(
```

Comment:
mentioned on discord, but can we export a balances object like this rather than using totalValue() and toUSDTBalances()

Reply:
I've tried, but we lose significant TVL due to pricing issues of curve tokens currently, this issue could affect other tokens in the future as well, therefore, if it's possible to merge the PR as is, please do so.

Reply:
hi, the issue here is, it is very hard for us to maintain the adapter with a lot of code, if some tokens are not priced, can you list them here and we will fix our coins api

Reply:
Hello, can you check if the price is correctly fetched for 0xbFcF63294aD7105dEa65aA58F8AE5BE2D9d0952A please, right now I think that's the only one with a problem